muxplore/main.c

```diff
@@ -2324,10 +2324,7 @@ int main(int argc, char *argv[]) {
     pthread_join(gen_item_thread, NULL);
     init_footer_elements();
     if (ui_count > 0) {
-        if (ui_count > theme.MUX.ITEM.COUNT) {
-            lv_obj_t * last_item = lv_obj_get_child(ui_pnlContent, -1);
-            lv_obj_set_height(last_item, lv_obj_get_height(last_item) + 50); // Don't bother asking...
```

Comment:
lv_obj_get_height(last_item) can return 0 in certain scenarios cause the height to get set to 50px.  In my case my items where 54px tall so this was actually causing the last 4px of the last item in the list to get cut off.

I found that instead of doing this what you want to do is call the following line after all items have been added ui_pnlContent.

lv_obj_update_layout(ui_pnlContent);

This will make sure that all items are displayed correctly when scrolling.